lib/linguist/languages.yml

```diff
@@ -2090,13 +2090,40 @@ JSON:
   - composer.lock
   - mcmod.info
   language_id: 174
-JSON5:
+JSON with comments:
   type: data
+  group: JSON
+  tm_scope: source.js
+  ace_mode: javascript
+  codemirror_mode: javascript
+  codemirror_mime_type: text/javascript
+  aliases:
+  - jsonc
   extensions:
-  - ".json5"
+  - ".jsonc"
```

Comment:
We're [a bit low on the `.json` usage](https://github.com/search?q=extension%3Ajsonc+NOT+dkfgjdkfjd)...

Reply:
Oops, you're right. :sweat_smile: Forgot to check the usage of `jsonc`.

Fixed!